src/inmanta/main.py

```diff
@@ -797,29 +807,36 @@ def version_report(client: Client, environment: str, version: str, show_detailed
 def monitor_deploy(client: Client, environment: str) -> None:
     tid = client.to_environment_id(environment)
 
-    versions = cast(dict[str, list[dict[str, str]]], client.do_request("list_versions", arguments=dict(tid=tid)))
-    allversion = versions["versions"]
-    try:
-        first: dict[str, str] = next(version for version in allversion if version["result"] != "pending")
-    except StopIteration:
-        raise click.ClickException("Environment %s doesn't contain a released configuration model" % environment)
+    def get_stats() -> tuple[int, int]:
+        deploy_status = client.do_request("resource_list", arguments=dict(tid=tid, deploy_summary=True, limit=0))
+
+        summary = deploy_status["metadata"]["deploy_summary"]
+        # {'by_state': {'available': 3, 'cancelled': 0, 'deployed': 12, 'deploying': 0, 'failed': 0, 'skipped': 0,
+        #               'skipped_for_undefined': 0, 'unavailable': 0, 'undefined': 0}, 'total': 15}
```

Comment:
This doesn't feel right at a glance, probably only relevant for the resource container in tests

Reply:
You mean the comment? I'll remove it